apps/studio/src/server/modules/folder/folder.router.ts

```diff
@@ -202,6 +232,21 @@ export const folderRouter = router({
               throw err
             })
 
+          // NOTE: update the index page's title so that they stay in sync
+          if (title) {
```

Comment:
question(non-blocking): hmm seems like the guard clause is not needed here? `title` is a required field so it should always be defined?

Reply:
ah good catch, i put it defensively without checking the type